Software/Python/line_follower/line_sensor.py

```diff
@@ -34,33 +34,22 @@
 # the integer division and input()
 # mind your parentheses!
 
-
-import smbus
 import time
 import math
 import RPi.GPIO as GPIO
 import struct
 import operator
 import pickle
 import numpy
+from periphery import I2C, I2CError
```

Comment:
Small request here.  Can you put a comment explaining why we switched to this, and put a link into this PR since you wrote a detailed explanation about what you found (https://github.com/DexterInd/GoPiGo/pull/273/) and maybe also link to the original firmware bug report (https://github.com/raspberrypi/firmware/issues/828) so if we ever have to pick this issue apart again, we have this info a little more readily accessible.

Reply:
Done.